app/models/claim/advocate_claim.rb

```diff
@@ -72,7 +72,7 @@ class AdvocateClaim < BaseClaim
              dependent: :destroy,
              inverse_of: :claim,
              validate: proc { |claim| claim.step_validation_required?(:basic_fees) }
-    has_many :fixed_fees,
+    has_many :fixed_fees, -> { order(:created_at) },
```

Comment:
❓ Changing the default order of the relation might affect how some of the pages during the claim submission process, or the claim summary page display the fees. Did we verify there was no impact there?